pages/orgs/[orgId]/settings.tsx

```diff
@@ -242,55 +294,67 @@ export default function EditOrgPage() {
         <main className={styles.main}>
           {userRole === "ADMIN" && (
             <div>
-              <h1>Edit Organisation</h1>
-              <form id="orgForm" onSubmit={submitHandler} className="column">
-                <TextField
-                  required
-                  label="Name"
-                  id="name"
-                  value={orgName}
-                  onChange={(event) => setOrgName(event.target.value)}
-                />
-                <Button
-                  variant="contained"
-                  type="submit"
-                  className="marginTopMedium"
-                >
-                  update
-                </Button>
+              <h1 className="text-3xl font-bold text-center">
```

Comment:
I believe we should use the [Heading](https://chakra-ui.com/docs/components/heading/usage) of chakra UI instead of custom styling. Any reason not do that? If yes, please add comments